components/net/file_loader.rs

```diff
@@ -96,8 +98,13 @@ pub fn factory(load_data: LoadData,
                             }
                         };
                     }
-                    Err(e) => {
-                        send_error(url, e.description().to_owned(), senders);
+                    Err(_) => {
+                        // this should be one of the three errors listed in
+                        // http://doc.rust-lang.org/std/fs/struct.OpenOptions.html#method.open
+                        // but, we'll go for a "404 Not Found!"
+                        let url = Url::parse("about:not-found").unwrap();
+                        let load_data_404 = LoadData::new(url, None);
+                        about_loader::factory(load_data_404, senders, classifier, cancel_listener)
```

Comment:
@jdm I'm confused what went wrong. I can see my changes, but reviewable can't see it.